scripts/ocr_evaluation.py

```diff
@@ -1,104 +1,124 @@
 import io
+import json
 import matplotlib.patches as patches
 import matplotlib.pyplot as plt
 import os
 import requests
 
 from PIL import Image
 
-from supercontrast.client import supercontrast_client
-from supercontrast.provider import Provider
-from supercontrast.task import OCRRequest, OCRResponse, Task
-
-# Define providers
+from supercontrast import (
+    OCRRequest,
+    OCRResponse,
+    Provider,
+    SuperContrastClient,
+    Task,
+    TaskMetadata,
+)
+
+# Constants
+TEST_IMAGE_URLS = [
+    "https://jeroen.github.io/images/testocr.png",
+    # Add more image URLs here
+]
 
 PROVIDERS = [
+    Provider.API4AI,
     Provider.AWS,
-    Provider.GCP,
     Provider.AZURE,
-    Provider.SENTISIGHT,
     Provider.CLARIFAI,
-    Provider.API4AI,
-    # Provider.OPENAI,
+    Provider.GCP,
+    Provider.SENTISIGHT,
 ]
 
-# Define image URLs to evaluate
+OUTPUT_DIR = "test_data/ocr"
 
-IMAGE_URLS = [
-    "https://jeroen.github.io/images/testocr.png",
-    # Add more image URLs here
-]
+# Helper functions
 
-# Define output directory for saving plots
 
-OUTPUT_DIR = "test_data/ocr"
+def print_request_response_and_metadata(
```

Comment:
move this function into utils?

Reply:
maybe make it nice and generic so that we aren't hardcoding the task name

Reply:
Fixed